<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>374</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">DeaDBeeF</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/root/images/deadbeef.png</normaloff>:/root/images/deadbeef.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <widget class="QMenu" name="loopingMenu">
     <property name="title">
      <string>Repeat</string>
     </property>
     <addaction name="actionLoopAll"/>
     <addaction name="actionLoopTrack"/>
     <addaction name="actionLoopNothing"/>
    </widget>
    <widget class="QMenu" name="shuffleMenu">
     <property name="title">
      <string>Shuffle</string>
     </property>
     <addaction name="actionNoShuffle"/>
     <addaction name="actionTrackShuffle"/>
     <addaction name="actionAlbumShuffle"/>
     <addaction name="actionRandomTrackShuffle"/>
    </widget>
    <addaction name="shuffleMenu"/>
    <addaction name="loopingMenu"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFiles"/>
    <addaction name="separator"/>
    <addaction name="actionAddFiles"/>
    <addaction name="actionAddFolder"/>
    <addaction name="actionAddURL"/>
    <addaction name="actionAddAudioCD"/>
    <addaction name="separator"/>
    <addaction name="actionNewPlaylist"/>
    <addaction name="actionLoadPlaylist"/>
    <addaction name="actionSaveAsPlaylist"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClearAll"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionDeselectAll"/>
    <addaction name="actionFind"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionBlockToolbarChanges"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuPlayback"/>
   <addaction name="Help"/>
  </widget>
  <action name="actionAddFolder">
   <property name="text">
    <string>Add folder(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next track</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionPrev">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous track</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNoShuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionTrackShuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tracks</string>
   </property>
  </action>
  <action name="actionAlbumShuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Albums</string>
   </property>
  </action>
  <action name="actionLoopAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All Tracks</string>
   </property>
  </action>
  <action name="actionLoopTrack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>One Track</string>
   </property>
  </action>
  <action name="actionLoopNothing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+P</string>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="text">
    <string>Add file(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAddAudioCD">
   <property name="text">
    <string>Add Audio CD</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionAddURL">
   <property name="text">
    <string>Add location</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+U</string>
   </property>
  </action>
  <action name="actionSaveAsPlaylist">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save playlist</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoadPlaylist">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Load playlist</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+L</string>
   </property>
  </action>
  <action name="actionNewPlaylist">
   <property name="text">
    <string>New playlist</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+N</string>
   </property>
  </action>
  <action name="actionHideMenuBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Main menu</string>
   </property>
   <property name="toolTip">
    <string>Show/hide main menu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionBlockToolbarChanges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Design mode</string>
   </property>
  </action>
  <action name="actionHideStatusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Statusbar</string>
   </property>
   <property name="toolTip">
    <string>Show/hide statusbar</string>
   </property>
  </action>
  <action name="actionClearAll">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+A</string>
   </property>
  </action>
  <action name="actionDeselectAll">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deselect all</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionFind">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSelectCurrentTrack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cursor follows playback</string>
   </property>
  </action>
  <action name="actionGoToCurrentTrack">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Jump to current track</string>
   </property>
  </action>
  <action name="actionScrollFollowsPlayback">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scroll follows playback</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionOpenFiles">
   <property name="text">
    <string>Open file(s)</string>
   </property>
  </action>
  <action name="actionRandomTrackShuffle">
   <property name="text">
    <string>Random Tracks</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
